public/index.html

```diff
@@ -7,6 +7,10 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
+  <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/stylesheets/components.css">
+  <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/landing.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
```

Comment:
I think CSS stylesheets' imports are preferred in CSS file iteself like App.css
Why are you you linking them in index, any specific reason?

Reply:
I have added it here to use preconnect with it  for faster loading, as css import is slow